packages/app-check/src/definitions.ts

```diff
@@ -118,11 +118,26 @@ export interface InitializeOptions {
    * @default false
    */
   isTokenAutoRefreshEnabled?: boolean;
+  /**
+   * Set used app check provider for Web.
+   *
+   * Only available for Web.
+   *
+   * Read more: https://firebase.google.com/docs/app-check/web/custom-provider
+   *
+   * @since 7.1.0
+   * @default ReCaptchaV3Provider
```

Comment:
```suggestion
   * The provider to use for App Check. Must be an instance of 
   * `ReCaptchaV3Provider`, `ReCaptchaEnterpriseProvider`, or `CustomProvider`.
   * 
   * Only available for Web.
   * 
   * @since 7.1.0
   * @see https://firebase.google.com/docs/app-check/web/custom-provider
```